interfaces/migrate/src/main/java/migrate/interfaces/JavaLib.java

```diff
@@ -0,0 +1,34 @@
+package migrate.interfaces;
+
+
+public class JavaLib implements Lib {
+    public String organization;
+    public String name;
+    public String revision;
+    public String crossVersion;
+
+    @Override
+    public String getCrossVersion() {
+        return crossVersion;
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public String getOrganization() {
+        return organization;
+    }
+    public String getRevision() {
+        return revision;
+    }
+
+    public JavaLib(String organization, String name, String revision, String crossVersion) {
+        this.organization = organization;
+        this.name = name;
+        this.revision = revision;
+        this.crossVersion = crossVersion;
+    }
```

Comment:
Being a data class which you use as key for a `Map`, you should consider implementing `equals` and `hashCode`
to use the attributes. Usually [this approach](https://mkyong.com/java/java-how-to-overrides-equals-and-hashcode/) is followed